src/models/heston.jl

```diff
@@ -0,0 +1,69 @@
+using QuadGK: quadgk
+
+"""
+[Heston model](https://en.wikipedia.org/wiki/Heston_model).
+"""
+@kwdef struct HestonModel{VT<:Real,VVT<:Real,KT<:Real,RT<:Real}
```

Comment:
I think we need a refactor to split the financial instruments defs from the models, but will address it in a follow-up since it will imply a large refactor.